### ISSUE TITLE
No Credentials Found Error When Deploying Package

### ISSUE THREAD

**Issue by adaje41** [daishi4u/ForceCode#22]
I have a ForceCode project with just two files, a package.xml and a destructiveChanges.xml, which I use to delete things from my org.  In the older version from celador, I would just do Deploy Package and it would push the package to the org and delete the files.

I recently updated to the daishi4u 3.5.3 version since the celador version didn't work with the latest VS Code.

Unfortunately since the update, I have not been able to deploy the destructive changes.  When I try, it would give me the error:

> Credential to salesforce server is not found in options.
Specify "username" and "password" in options, or give any other credentials.

Credentials are in the force.json though, and I can confirm I'm logged in to the org since I can do all other operations (Open File, Execute Anonymous, etc..)

**Comment by julesx** [comment]
password isn't meant to be specified in the force.json file anymore, only username, so i'm guessing this feature hasn't been properly ported from the celador master.

**Comment by daishi4u** [comment]
I'm working on this. I have got it to successfully log in with the current ForceCode connection, but now I'm getting "invalid uri /services/soap/m/39.0" every time. It seems like it's a jsforce issue, but I haven't figured out anything quite yet. Will keep you posted...

**Comment by daishi4u** [comment]
If anyone has used jsforce and knows how to fix "invalid uri /services/soap/m/39.0" I would appreciate some input. Thanks!

**Comment by daishi4u** [comment]
This should be fixed in v3.5.9 If you have any other problems with it then let me know, otherwise I will close this issue today. Thanks!

**Comment by adaje41** [comment]
Sorry for the late reply, got tied up on another project.  I tried this just now, still trying to deploy the destructiveChanges.xml and package.xml.  I'm getting the below error though:

> Metadata Describe Error. Please try again.

**Comment by daishi4u** [comment]
Ok, did you enable the handleMetaFiles option in force.json?

**Comment by adaje41** [comment]
Yes sir.  That menu option to Deploy Package doesn't even show up if handleMetaFiles is false.  So I have mine set to true.  (I'm using the Retrieve Package option for my other project, which also requires handleMetaFiles)

Anyways I tried again just today and I flipped handleMetaFiles back and forth just to make sure (restarting VSCode in between).  Interestingly I'm getting a different error today:

> ForceCode: Deploy Errors Element {http://soap.sforce.com/2006/04/metadata}verbose invalid at this location
